nucleus/slice.py

```diff
@@ -234,6 +234,38 @@ def items_and_annotations(
         )
         return convert_export_payload(api_payload[EXPORTED_ROWS])
 
+    def items_and_predictions(
```

Comment:
In general I think it would be easier to use if we had the same interface for exporting items, predictions and annotations between a `slice` and a `dataset`. Right now the corresponding method in a dataset is `export_predictions(model)`. I'd make this method consistent with the dataset one. 

Just wondering in case we have a huge export, wouldn't we also run into timeout issues then with this request?